draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1296,9 +1296,10 @@ Truncating:
 
 Error Handling:
 : If a system overruns the generator by requesting too many UUIDs
-  within a single system time interval, the UUID service SHOULD either
+  within a single system time interval, the UUID service can either
```

Comment:
Are only two possible actions allowed here, error or stalling? If not, I recommend removing the word "either" here.

Paraphrasing: If too many, maybe error or maybe stall; DO NOT duplicate.